src/tools/tracejob.h

```diff
@@ -94,6 +94,7 @@ struct log_entry
 {
 	char *date;           /* date of log entry */
 	time_t date_time;     /* number of seconds from the epoch to date */
+	long highres_ts;  /* high resolution time stamp */
```

Comment:
Please indent the comment. I think the variable name is misleading here. When we parse the log message microseconds are read into highres_ts which is part of a timestamp and not really a timestamp by itself.